testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/weld/modules/export/AlphaExtension.java

```diff
@@ -29,7 +29,7 @@
 public class AlphaExtension implements Extension {
 
     public void registerBravo(@Observes BeforeBeanDiscovery event, BeanManager manager) {
-        event.addAnnotatedType(manager.createAnnotatedType(BravoBean.class));
-        event.addAnnotatedType(manager.createAnnotatedType(CharlieBean.class));
+        event.addAnnotatedType(manager.createAnnotatedType(BravoBean.class), null);
```

Comment:
Just a side note, I'd recommend avoiding the use of `null` as an identifier. It "just works" because Weld happens to deal with it but this isn't specified anywhere and might change.
You can do something as simple as `BravoBean.class.getName()`

Reply:
👍  Thanks.